orttraining/orttraining/core/session/training_session.cc

```diff
@@ -655,6 +653,35 @@ void TrainingSession::AddPredefinedTransformers(GraphTransformerManager& transfo
   }
 }
 
+Status TrainingSession::ApplyModelParallelTransformationsToMainGraph(std::unordered_set<std::string>& weights_to_train,
```

Comment:
can this be const?

Reply:
weights_to_train cannot be const reference here. Because MegatronTransformer need update the name after weight partition.